k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java

```diff
@@ -288,8 +288,8 @@ public void onClick(View view) {
     }
 
     public void showCryptoDialog(CryptoMode currentCryptoMode) {
-        CryptoSettingsDialog dialog = CryptoSettingsDialog.newInstance(currentCryptoMode);
-        dialog.show(activity.getFragmentManager(), "crypto_settings");
+        //CryptoSettingsDialog dialog = CryptoSettingsDialog.newInstance(currentCryptoMode);
```

Comment:
will this then be an empty dialog?

Reply:
It is actually obsolete code from K-9 that is not used. So I just commented code that was hindering the refactor itself.

Reply:
I guess then we can remove the entire function as it does nothing

Reply:
alright, removed, thanks.